.pipelines/e2e-windows.yaml

```diff
@@ -49,6 +49,14 @@ parameters:
   displayName: Windows NV Series VM Size (gen2)
   type: string
   default: Standard_NV6ads_A10_v5
+- name: test23H2containerd
```

Comment:
I prefer to remove "containerd" that same as other varibales

Reply:
Fixed. Miss here before.